features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/EnabledErrorTypesScenario.m

```diff
@@ -32,10 +32,8 @@ - (void)startBugsnag {
 
 - (void)run {
     // From null prt scenario
-    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(2 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
-        volatile char *ptr = NULL;
-        (void) *ptr;
-    });
+    volatile char *ptr = NULL;
+    (void) *ptr;
```

Comment:
What are these for?

Reply:
This intentionally crashes the app when the scenario is run from the test script:

```cucumber
  Scenario: All Crash reporting is disabled
    # Sessions: on, unhandled crashes: off
    When I run "DisableAllExceptManualExceptionsAndCrashScenario" and relaunch the app

```

This particular test script then goes on to verify that the crash is not reported by the notifier.